src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -201,6 +221,7 @@ HttpStatusCode mongoSubscribeContext
     LM_T(LmtSubCache, ("inserting a new sub in cache (%s)", oidString.c_str()));
 
     cacheSemTake(__FUNCTION__, "Inserting subscription in cache");
+    LM_W(("Calling subCacheItemInsert with status == %s", status.c_str()));
```

Comment:
This seems to be a "regular" trace... why is this in warning level? Maybe a leftover?